Tests/docker_native_image_config.json

```diff
@@ -30,6 +30,36 @@
             "tesseract"
          ],
          "docker_ref": "demisto/py3-native:8.1.0.40949"
+      },
+      "native:dev":{
+         "supported_docker_images":[
+            "python3",
+            "python3-deb",
+            "python3-ubi",
+            "py3-tools",
+            "py3-tools-ubi",
+            "crypto",
+            "readpdf",
+            "parse-emails",
+            "docxpy",
+            "sklearn",
+            "pandas",
+            "ippysocks-py3",
+            "oauthlib",
+            "unzip",
+            "py3ews",
+            "taxii2",
+            "pan-os-python",
+            "slackv3",
+            "google-api-py3",
+            "boto3py3",
+            "pyjwt3",
+            "joe-security",
+            "slack",
+            "office-utils",
+            "chromium",
+            "tesseract"
+         ]
       }
```

Comment:
This should be done only when we have an SDK release cause it add it to the production bucket as well.